src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -2491,3 +2491,17 @@ instruct scatterL(vmemA mem, vReg src, vReg idx) %{
   %}
   ins_pipe(pipe_slow);
 %}
+
+// ------------------------------ Vector Load Const -------------------------------
+instruct loadconB(vReg dst, immI0 src)
+%{
+  predicate(UseSVE > 0 &&
+            n->bottom_type()->is_vect()->element_basic_type() == T_BYTE);
```

Comment:
When the size is 16, the neon pattern will also be matched. Can you please add "UseSVE == 0" for NEON VectorLoadConst patterns?

Reply:
Thank you for your review. I have changed that.